data/config/fidesops.toml

```diff
@@ -41,3 +41,4 @@ enabled = true
 
 [notifications]
 send_request_completion_notification = true
+send_request_receipt_notification = true
```

Comment:
Why are these true, but the base fidesops.toml has them as false? is this for testing?